cmd/host-profiler/dist/host-profiler-config.yaml

```diff
@@ -33,11 +46,12 @@ service:
   pipelines:
     profiles:
       receivers: [hostprofiler]
-      # infraattributes/default is automatically removed when the Agent Core is not available  
-      processors: [infraattributes/default]
-      exporters: [otlphttp, debug]   
+      # infraattributes/default is automatically removed when the Agent Core is not available
+      # resourcedetection is automatically removed when the Agent Core is available
+      processors: [infraattributes/default, resourcedetection]
```

Comment:
One thought just occurred to me. Since we always want to add it to get `host.arch` regardless of the customer configuration, I'm not sure what we should include in the default config file 💭  But we can leave figuring this out to a follow-up PR when we implement always adding `resourcedetection` / `k8sattributesprocessor` for standalone mode.